releases/release-1.20/release-notes-draft.md

```diff
@@ -1,178 +1,264 @@
+
+### Introducing Graceful Node Shutdown (Alpha)
+The `GracefulNodeShutdown` feature is now in Alpha. This allows kubelet to be aware of node system shutdowns, enabling graceful termination of pods during a system shutdown. This feature can be [enabled through feature gate](https://docs.k8s.io/concepts/architecture/nodes/#graceful-node-shutdown).
+
+### Runtime log sanitation
+Logs can now be configured to use runtime protection from leaking sensitive data. [Details for this experimental feature is available in documentation](https://docs.k8s.io/concepts/cluster-administration/system-logs/#log-sanitization).
+
+### Pod resource metrics
+On-demand metrics calculation is now available through `/metrics/resources`. [When enabled](
+https://docs.k8s.io/concepts/cluster-administration/system-metrics#kube-scheduler-metrics), the endpoint will report the requested resources and the desired limits of all running pods.
+
+### Introducing `RootCAConfigMap`
+`RootCAConfigMap` graduates to Beta, seperating from `BoundServiceAccountTokenVolume`. The `kube-root-ca.crt` ConfigMap is now available to every namespace, by default. It contains the Certificate Authority bundle for verify kube-apiserver connections.
+
+### `kubectl debug` graduates to Beta
+`kubectl alpha debug` graduates from alpha to beta in 1.20, becoming `kubectl debug`.
+`kubectl debug` provides support for common debugging workflows directly from kubectl. Troubleshooting scenarios supported in this release of `kubectl` include:
+Troubleshoot workloads that crash on startup by creating a copy of the pod that uses a different container image or command.
+Troubleshoot distroless containers by adding a new container with debugging tools, either in a new copy of the pod or using an ephemeral container. (Ephemeral containers are an alpha feature that are not enabled by default.)
+Troubleshoot on a node by creating a container running in the host namespaces and with access to the host’s filesystem.
+Note that as a new builtin command, `kubectl debug` takes priority over any `kubectl` plugin named “debug”. You will need to rename the affected plugin.
+Invocations using `kubectl alpha debug` are now deprecated and will be removed in a subsequent release. Update your scripts to use `kubectl debug` instead of `kubectl alpha debug`!
+For more information about kubectl debug, see Debugging Running Pods on the Kubernetes website, kubectl help debug, or reach out to SIG CLI by visiting #sig-cli or commenting on [enhancement #1441](https://features.k8s.io/1441).
+
+### Removing deprecated flags in kubeadm
+`kubeadm` removes a significant number of deprecated flags in this release. The complete list of flags are available below on Urgent Upgrade Notes section.
```

Comment:
i'd reword this section to something in the lines of the following:

```
### Deprecations and deprecated feature removals in kubeadm
`kubeadm` applies a number of deprecations and removals of deprecated features in this release. More details are available in the Urgent Upgrade Notes and Kind / Deprecation sections.
```

rationale:
- the most important deprecation is the "master" label/taint.
- the deprecations and removals are spread across a couple of different sections.

Reply:
sounds good to me, thanks for reviewing @neolit123!